README.md

```diff
@@ -9,8 +12,8 @@ ensure your application behaves appropriately under harsh conditions. Before you
 can use the Ruby library, you need to read the [Usage section of the Toxiproxy
 README](https://github.com/shopify/toxiproxy#usage).
 
-```
-gem install toxiproxy
+```shell
+$ gem install toxiproxy
```

Comment:
Super minor, but I am not a fan of including `$` in command output unless we need to distinguish between typed commands and output being presented to the user.

Advantage of not including the `$` is that it makes it easier for people to copy pasta the actual command they need (there's a handy "copy this text" button on Github Markdown boxes) without having to manually remove the preceding `$`.

Reply:
> is that it makes it easier for people to copy pasta

that exactly the reason why I am adding $ to remove people think before copy paste. and show that it is command line.

Reply:
Ok. Aleem on my side :P